how-to-docs/namespace-how-to.md

```diff
@@ -19,5 +19,5 @@ metadata:
 ```
 to be defined in `namespaces/bar/foo.yaml`
```

Comment:
`policies/bar/namespaces/foo.yaml`

Reply:
does this need to be policies/clusters/bar/namespaces/foo.yaml